src/main/java/net/logstash/logback/composite/FormattedTimestampJsonProvider.java

```diff
@@ -204,11 +204,60 @@ public void setPattern(String pattern) {
         this.pattern = pattern;
         updateTimestampWriter();
     }
+    
+    /**
+     * Get the time zone used to write the timestamp.
+     * 
+     * @return the time zone used to write the timestamp
+     */
     public String getTimeZone() {
         return timeZone.getID();
     }
+    
+    
+    /**
+     * Set the timezone for which to write the timestamp.
+     * Only applicable if the pattern is not {@value #UNIX_TIMESTAMP_AS_NUMBER} or {@value #UNIX_TIMESTAMP_AS_STRING}.
+     * 
+     * <p>The TimeZone can be expressed into any format supported by {@link TimeZone#getTimeZone(String)}.
+     * It can be a valid time zone ID. For instance, the time zone ID for the
+     * U.S. Pacific Time zone is "America/Los_Angeles".
+     * 
+     * <p>If the time zone you want is not represented by one of the
+     * supported IDs, then a custom time zone ID can be specified to
+     * produce a TimeZone. The syntax of a custom time zone ID is:
+     *
+     * <blockquote><pre>
+     * <i>CustomID:</i>
+     *         <code>GMT</code> <i>Sign</i> <i>Hours</i> <code>:</code> <i>Minutes</i>
+     *         <code>GMT</code> <i>Sign</i> <i>Hours</i> <i>Minutes</i>
+     *         <code>GMT</code> <i>Sign</i> <i>Hours</i>
+     * <i>Sign:</i> one of
+     *         <code>+ -</code>
+     * <i>Hours:</i>
+     *         <i>Digit</i>
+     *         <i>Digit</i> <i>Digit</i>
+     * <i>Minutes:</i>
+     *         <i>Digit</i> <i>Digit</i>
+     * <i>Digit:</i> one of
+     *         <code>0 1 2 3 4 5 6 7 8 9</code>
+     * </pre></blockquote>
+     *
+     * <i>Hours</i> must be between 0 to 23 and <i>Minutes</i> must be
+     * between 00 to 59.  For example, "GMT+10" and "GMT+0010" mean ten
+     * hours and ten minutes ahead of GMT, respectively.
+     * 
+     * <p>Use a blank string or {@code null} to use the default TimeZone of the system.
```

Comment:
Joran does not allow to set a blank string nor a null value... May be we need another way here for the user to say it wants the default time zone of the system... ?

Reply:
Adding a special case for "DEFAULT" seems ok to me.

Reply:
`[DEFAULT]` can now be used to refer to the system default time zone (between brackets to follow same convention as used by the `pattern` property of the same class).